common/scala/src/main/scala/org/apache/openwhisk/core/entity/Identity.scala

```diff
@@ -129,10 +131,12 @@ object Identity extends MultipleReadersSingleWriterCache[Option[Identity], DocIn
       stale = StaleParameter.No)
   }
 
-  private def rowToIdentity(row: JsObject, key: String)(implicit transid: TransactionId, logger: Logging) = {
+  protected[entity] def rowToIdentity(row: JsObject, key: String)(implicit transid: TransactionId, logger: Logging) = {
     row.getFields("id", "value", "doc") match {
       case Seq(JsString(id), JsObject(value), doc) =>
-        val limits = Try(doc.convertTo[UserLimits]).getOrElse(UserLimits())
+        val limits =
+          if (doc != JsNull) Try(doc.convertTo[UserLimits]).getOrElse(UserLimits.standardUserLimits)
```

Comment:
Does this imply the deafult value for `UserLimits` may change in the future so that you use a variable instead of direct `UserLimits()`?